frontends/mit-open/src/pages/HomePage/carousels.ts

```diff
@@ -0,0 +1,70 @@
+import type { ResourceCarouselProps } from "@/page-components/ResourceCarousel/ResourceCarousel"
+import type { FeaturedApiFeaturedListRequest as FeaturedListParams } from "api"
+
+const FEATURED_COMMON_PARAMS: FeaturedListParams = {
+  resource_type: ["course"],
+  limit: 12,
+}
+const FEATURED_RESOURCES_CAROUSEL: ResourceCarouselProps["config"] = [
+  {
+    label: "All",
+    cardProps: { size: "medium" },
+    data: {
+      type: "lr_featured",
+      params: { ...FEATURED_COMMON_PARAMS },
+    },
+  },
+  {
+    label: "Free",
+    cardProps: { size: "medium" },
+    data: {
+      type: "lr_featured",
+      params: { ...FEATURED_COMMON_PARAMS, free: true },
+    },
+  },
+  {
+    label: "Certificate",
+    cardProps: { size: "medium" },
+    data: {
+      type: "lr_featured",
+      params: { ...FEATURED_COMMON_PARAMS, certification: true },
+    },
+  },
+  {
+    label: "Professional and Executive Education",
```

Comment:
"Professional & Executive Learning" , just because that's what it says here https://prolearn.mit.edu/